geomstats/tests.py

```diff
@@ -155,19 +109,21 @@ def assertAllClose(self, a, b, rtol=gs.rtol, atol=gs.atol):
             msg=pytorch_error_msg(a, b, rtol, atol),
         )
 
+    def assertTrue(self, condition, msg=None):
+        assert condition, msg
+
+    def assertFalse(self, condition, msg=None):
+        assert not condition, msg
+
+    def assertEqual(self, a, b):
+        assert a == b
+
     def assertAllCloseToNp(self, a, np_a, rtol=gs.rtol, atol=gs.atol):
         are_same_shape = np.all(a.shape == np_a.shape)
         are_same = np.allclose(a, np_a, rtol=rtol, atol=atol)
-        if tf_backend():
-            return super().assertTrue(are_same_shape and are_same)
-        return super().assertTrue(are_same_shape and are_same)
+        assert are_same and are_same_shape == True
```

Comment:
I don't think you need == True at the end here

Reply:
correct